### PR TITLE
2.3.0-changelog

### PR DESCRIPTION
**Pull request by songawee**
connect https://github.com/waffleio/waffle.io-takeout/issues/145

### CODE REVIEW

File: releases/2.x.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Waffle Takeout 2.2
 
+## Waffle Takeout 2.3.0
+
```
Comment (homeyer) [inline releases/2.x.md]: can you add the release date?

Comment (songawee) [inline releases/2.x.md]: on it

**Comment by homeyer** [comment]
👍

File: releases/2.x.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+_Released 2016.06.07_
+
+_Supported GH:E versions: GH:E 2.2+_
+
+#### Features
+- Package MongoDB instance within Takeout
+- Enable users to install Takeout without needing to open up ports 3000-3009 on the host machine
+
+#### Bug Fixes
+- Join link is broken
```
Comment (burtonjc) [inline releases/2.x.md]: Do we want to state these in the positive? Like "Join link is fixed"?

**Pull request merged**